dashboard/app/models/course_version.rb

```diff
@@ -43,12 +45,7 @@ def units
     content_root.default_units
   end
 
-  # "Interface" for content_root:
-  #
-  # is_course? - used during seeding to determine whether this object represents the content root for a CourseVersion.
-  #   For example, this should return True for the CourseA-2019 Unit and the CSP-2019 UnitGroup. This should return
-  #   False for the CSP1-2019 Unit.
-  belongs_to :content_root, class_name: "UnitGroup", optional: true
+  belongs_to :content_root, polymorphic: true, optional: true
```

Comment:
even though you reverted the DB migration, I think you should still change this to `belongs_to`. this is necessary in order to make the code stop depending on the content_root_type field, ahead of the migration which removes the column from the DB.

Reply:
sorry -- I meant, you should get rid of polymorphic: true, which is a hidden code dependency on the content_root_type column.